src/promptflow/promptflow/_sdk/_service/apis/collector.py

```diff
@@ -0,0 +1,31 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# this file is different from other files in this folder
+# functions (APIs) defined in this file follows OTLP 1.1.0
+# https://opentelemetry.io/docs/specs/otlp/#otlphttp-request
+# to provide OTLP/HTTP endpoint as OTEL collector
+
+from flask import request
+from opentelemetry.proto.collector.trace.v1.trace_service_pb2 import ExportTraceServiceRequest
+
+from promptflow._sdk.entities._trace import Span
+
+
+def trace_collector():
+    content_type = request.headers.get("Content-Type")
+    # binary protobuf encoding
+    if "application/x-protobuf" in content_type:
```

Comment:
may add a simple test so it's easier for others to understand how to call this rest api. 
Then we can merge once.

Reply:
will there be any manually-created request to this endpoint? I understand all network traffic comes from OTLP exporter?

Reply:
add an example